app.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line no-unused-vars
+function togglePopup() {
+  // eslint-disable-next-line indent
+  document.getElementById('popup-1').classList.toggle('active');
+}
+
+// eslint-disable-next-line no-undef
```

Comment:
- [ ] **Required:** Kindly fix errors rather than disable them. Linters provide a standard rule for community/team codebases and it is good practice and learn a programming languages style guide.